src/client/components/ranking-bar/index.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import './index.scss';
+import {Menu, Dropdown} from 'semantic-ui-react';
+
+
+export class RankingBarComponent extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            activeItem: 'price'
```

Comment:
Please create a constant for that, right in this file. You can define it like this:
```javascript
const ITEM_NAME = {
    PRICE: 'price',
    DISTANCE: 'distance',
    // ...
};

// Then use it like this:

const x = ITEM_NAME.PRICE;
```